code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -215,11 +220,12 @@
 
 /obj/item/reagent_containers/hypospray/medipen/survival
 	name = "survival medipen"
-	desc = "A medipen for surviving in the harshest of environments, heals and protects from environmental hazards. WARNING: Do not inject more than one pen in quick succession."
+	desc = "A medipen for surviving in the harshest of environments, heals and protects from environmental hazards. WARNING: Do not inject more than one pen in quick succession. Due to the nature of the chemicals inside, this injector will only work in low pressure environments."

```

Comment:
this desc doesnt make much sense since you can still use the reagents on themselves just not with the pen

Reply:
Good point ! I'll change it to mention the mechanism.